drivers/flash/flash_stm32h7x.c

```diff
@@ -33,7 +33,9 @@ LOG_MODULE_REGISTER(LOG_DOMAIN);
 /* Let's wait for double the max erase time to be sure that the operation is
  * completed.
  */
-#define STM32H7_FLASH_TIMEOUT (2 * DT_PROP(DT_INST(0, st_stm32_nv_flash), max_erase_time))
+#define STM32H7_FLASH_TIMEOUT     (2 * DT_PROP(DT_INST(0, st_stm32_nv_flash), max_erase_time))
+/* No information in documentation about that. */
+#define STM32H7_FLASH_OPT_TIMEOUT (800)
```

Comment:
It would be nice to indicate that this timeout is in milliseconds. How about `STM32H7_FLASH_OPT_TIMEOUT_MS` ? I also think that parentheses are not necessary.

Reply:
Done.